__tests__/pages/dashboard/tracing/[caseId]/[contactId]/monitor.test.tsx

```diff
@@ -19,4 +19,89 @@ describe("Monitor", () => {
       "/dashboard/tracing/undefined/undefined"
     );
   });
+
+  it("renders `Informasi Utama` section corectly", () => {
+    render(<Monitor />);
+    expect(screen.getByText(/informasi utama/i));
+  });
+
+  it("render `Jenis Karantina` section corectly", () => {
+    render(<Monitor />);
+    expect(screen.getByText(/jenis karantina/i));
+    expect(screen.getByRole("button", { name: /tidak karantina/i }));
+    fireEvent(
+      screen.getByRole("button", { name: /tidak karantina/i }),
+      new MouseEvent("click", {
+        bubbles: true,
+        cancelable: true,
+      })
+    );
```

Comment:
Based on our [testing guidelines](https://kcov.id/testing-guidelines), please refactor all event triggers using `fireEvent` into interaction triggers using `userEvent` instead.
You can read more about it in the [Interactions vs. events](https://testing-library.com/docs/guide-events/#interactions-vs-events) guide here.

Reply:
okey. thank you for review my code.
fix it asap.